File: beecrowd/1/1018/1018.java
import java.io.IOException;
import java.util.Scanner;
import java.lang.Math;

class Main {

    public static void main(String[] args) throws IOException {

        Scanner sc = new Scanner(System.in);

        int dinheiro, quantidade;
        dinheiro = sc.nextInt();

        int notas[] = {100, 50, 20, 10, 5, 2, 1};
        System.out.printf("%d\n", dinheiro);
        for (int i = 0; i < 7; i++) {
            quantidade = dinheiro / notas[i];
            dinheiro -= quantidade * notas[i];
            System.out.printf("%d nota(s) de R$ %d,00\n", quantidade, notas[i]);
        }

        sc.close();

    }

}

File: beecrowd/1/1018/1018.py
dinheiro = int(input())
quantidade = 0

notas = [100, 50, 20, 10, 5, 2, 1]

print(dinheiro)

for i in range(len(notas)):
    quantidade = dinheiro // notas[i]
    dinheiro -= quantidade * notas[i]
    print("%d nota(s) de R$ %d,00" % (quantidade, notas[i]))
